frontend/src/metabase/query_builder/components/QueryVisualization.jsx

```diff
@@ -30,6 +31,7 @@ export default function QueryVisualization(props) {
     maxTableRows = HARD_ROW_LIMIT,
   } = props;
 
+  const canRun = useMemo(() => Lib.canRun(question.query()), [question]);
```

Comment:
I remember we optimized that part with canRun. Why can't we avoid using memo?

Reply:
Agree, removed useMemo